src/WebJobs.Script/Binding/Extensibility/ScriptBindingContext.cs

```diff
@@ -112,10 +112,9 @@ public ScriptBindingContext(JObject bindingMetadata)
         /// <returns>The metadata value.</returns>
         public TValue GetMetadataValue<TValue>(string name, TValue defaultValue = default(TValue))
         {
-            JToken value = null;
             try
             {
-                if (Metadata.TryGetValue(name, StringComparison.OrdinalIgnoreCase, out value))
+                if (Metadata.TryGetValue(name, StringComparison.OrdinalIgnoreCase, out var value))
```

Comment:
can this be `JToken` rather than `var`?

Reply:
Good catch, this ended up being refactored in the process but I meant to keep the type explicit